### PR TITLE
Update scaling-eth.md

### PR DESCRIPTION
**Pull request by adrianbrink**

### CODE REVIEW

File: md/scaling-eth.md [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
 
-TODO: Write about the different paths to scaling Ethereum DAPPS using our stack.
+In the following we will briefly describe different approaches to scaling Ethereum.
+
 
 ## Ethermint
-Use Ethermint...
+The first approach is to use Ethermint to deploy your Dapps. Ethermint is a PoS
+implementation of the Ethereum virtual machine. It implements the Web3 standard
+and hence all of the tooling, such as wallets and UIs, build for Ethereum works
```
Comment (jolesbi) [inline md/scaling-eth.md]: "build" or "built"?

File: md/scaling-eth.md [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+Cosmos Hub. On the Cosmos Hub they can be exchanged at a much higher rate. Furthermore
+the hub allows the movement of those tokens to specialised payment or EVM zones,
+where users can use their tokens. The peg zone allows any user to withdraw their
+tokens back to Ethereum in a permissionless way.
 
-## Plasma
-Use our implementation of Plasma...
+## Tendermint Plasma
+Tendermint Plasma allows projects to build their dedicated blockchains applications
+using the Cosmos-Sdk, while their tokens are staked on Ethereum. This means that
```
Comment (jolesbi) [inline md/scaling-eth.md]: Sdk or SDK?

Comment (jolesbi) [reply md/scaling-eth.md]: what's the different between Ethermint and Tendermint Plasma? they sound very similar.

File: md/scaling-eth.md [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+projects don't have to adapt to a completely new security model, but can rather
+rely on the security properties of Ethereum, while benefiting from much higher
+transaction throughput on the Tendermint child chain.
 
 ## Cosmos SDK
```
Comment (jolesbi) [inline md/scaling-eth.md]: what does this have to do with scaling-eth?

File: md/scaling-eth.md (hunk 2 above)
Comment (nylira) [reply md/scaling-eth.md]: Ethermint is a one for one drag and drop replacement for Ethereum, Tendermint Plasma will achieve scaling by ditching the EVM for a more performant alternative - Cosmos SDK

File: md/scaling-eth.md (hunk 1 above)
Comment (nylira) [reply md/scaling-eth.md]: Fixed it
